update.c

```diff
@@ -91,8 +92,13 @@ int main(int argc, char const *argv[]) {
       fprintf(stderr, "%s\n", error.message);
     } else {
       printf("Document edited!\n");
-      str = bson_as_canonical_extended_json(update, NULL);
```

Comment:
The current code was just printing the `update` document which we pass to `mongoc_collection_update_one` , and not the updated document from the collection.
New code will fetch the doc from the collection and then print it.